main.go

```diff
@@ -127,6 +120,29 @@ func (c *Collector) extractMetrics(ch chan<- prometheus.Metric, rm resourceMeta,
 	}
 }
 
+func getAliasForMetricName(metricName string) string {
+	switch metricName {
+	default:
```

Comment:
Strange upside-down construction :) 
Any reason why default goes first?

Reply:
To notice that alias for default is same like name. Anyway changed to have same code style like everywhere else :).